package.json

```diff
@@ -34,13 +34,16 @@
   "scripts": {
     "tsc": "tsc",
     "lint": "tslint src/**/*.ts",
-    "install": "node scripts/install.js",
-    "postinstall": "node scripts/post-install.js",
+    "install": "prebuild-install || prebuild --install",
+    "rebuild": "prebuild --compile",
     "test": "cross-env NODE_ENV=test mocha -R spec lib/*.test.js",
+    "prebuild": "node scripts/prebuild.js",
     "prepublish": "npm run tsc"
   },
   "dependencies": {
-    "nan": "^2.6.2"
+    "nan": "^2.6.2",
+    "prebuild": "^7.4.0",
```

Comment:
Are both of these dependencies or is one a dev dependency?

Reply:
Both need to be regular dependencies. The prebuild program is a fallback for those on some unsupported platform.